frontend/packages/volto-intranet/src/components/Rating/Rating.jsx

```diff
@@ -0,0 +1,222 @@
+import { Button, Container } from '@plone/components';
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { Link, useLocation } from 'react-router-dom';
+import Icon from '@plone/volto/components/theme/Icon/Icon';
+import useUser from '@plone/volto/hooks/user/useUser';
+import cx from 'classnames';
+import { defineMessages, useIntl } from 'react-intl';
+import { toast } from 'react-toastify';
+import Toast from '@plone/volto/components/manage/Toast/Toast';
+import { toogleLike } from '../../actions/likes/likes';
+import { flattenToAppURL } from '@plone/volto/helpers/Url/Url';
+import thumbsSVG from '../../icons/icon-thumbs.svg';
+import thumbsFilledSVG from '../../icons/icon-thumbs-filled.svg';
+import commentSVG from '../../icons/comment.svg';
+import shareSVG from '../../icons/share.svg';
+import FormattedDate from '@plone/volto/components/theme/FormattedDate/FormattedDate';
+
+const messages = defineMessages({
+  loginToLike: {
+    id: 'To submit a Like you must be logged in',
+    defaultMessage: 'To submit a Like you must be logged in',
+  },
+  modified: {
+    id: 'Last Modified On',
+    defaultMessage: 'Last Modified On',
+  },
+  created: {
+    id: 'Created On',
+    defaultMessage: 'Created On',
+  },
+  share: {
+    id: 'Share Content',
+    defaultMessage:
+      'Dear, I would like to share the following News with you: {link} Kind regards {fullname}',
+  },
+  likeFailed: {
+    id: 'Something went wrong while liking this post.',
+    defaultMessage: 'Something went wrong while liking this post.',
+  },
+  unlikeFailed: {
+    id: 'Something went wrong while unliking this post.',
+    defaultMessage: 'Something went wrong while unliking this post.',
+  },
+});
+const DotFormattedDate = ({ date, className, locale }) => {
+  return (
+    <FormattedDate
+      className={className}
+      date={date}
+      format={{
+        year: 'numeric',
+        month: '2-digit',
+        day: '2-digit',
+      }}
+      locale={locale}
+    >
+      {(parts) => {
+        const day = parts.find((p) => p.type === 'day')?.value;
+        const month = parts.find((p) => p.type === 'month')?.value;
+        const year = parts.find((p) => p.type === 'year')?.value;
+        return `${day}.${month}.${year}`;
+      }}
+    </FormattedDate>
```

Comment:
I'm not sure about the date format; @sneridagh shouldn't we use the normal short date format for the current locale, instead of a custom one? (I know this is matching the design; I'm saying the format shown in the design won't make sense in every language.)